fip-0018.md

```diff
@@ -93,9 +93,48 @@ No changes
 ##### Example
 No changes
 
+### Remove Public Address
+#### Existing action name: *remaddress*
+#### Existing end point: [/remove_pub_address](https://github.com/fioprotocol/fips/blob/master/fip-0004.md)
+#### Request
+No changes
+##### Example
+```
+{
+  "fio_address": "purse@alice",
+  "public_addresses": [
+    {
+      "chain_code": "BTC",
+      "token_code": "BTC",
+      "public_address": "1PMycacnJaSqwwJqjawXBErnLsZ7RkXUAs"
+    },
+    {
+      "chain_code": "ETH",
+      "token_code": "*",
+      "public_address": "0xab5801a7d398351b8be11c439e05c5b3259aec9b"
+    }
+  ],
+  "max_fee": 0,
+  "tpid": "rewards@wallet",
+  "actor": "aftyershcu22"
+}
+```
+#### Processing
+When * is specified as token_code only that entry is removed. For example is FIO Address is mapped to ETH-USDC and ETH-* and user removes ETH-* as a mapping, ETH-USDC is retained.
+#### Exception handling
+Specifying * as token code will now be allowed.
```

Comment:
Confused by this as the previous comment says "When * is specific as a token_code"

Reply:
Which part is confusing?